### PR TITLE
Update Enchanting inventory to include new slot

### PR DESCRIPTION
**Pull request by ZephireNZ**
Adds a couple convenience methods to EnchantingInventory due to the addition of the 'Lapis' slot, and update the InventoryType to include the new slot.

### CODE REVIEW

File: src/main/java/org/bukkit/event/inventory/InventoryType.java [hunk 1]
```diff
@@ -30,10 +30,10 @@
      */
     CRAFTING(5,"Crafting"),
     /**
-     * An enchantment table inventory, with one CRAFTING slot and three
+     * An enchantment table inventory, with 2 CRAFTING slots and three
```
Comment (turt2live) [inline src/main/java/org/bukkit/event/inventory/InventoryType.java]: If you want to get really picky, proper English rules for documentation state that this particular number should be written in it's word form.

Comment (ZephireNZ) [inline src/main/java/org/bukkit/event/inventory/InventoryType.java]: I went with the number form, as almost all of the other documentation also do it that way. If you want I can change it, but it's not like it's a huge problem.

**Comment by SpaceManiac** [comment]
There was some discussion in IRC about the name `getLapis()`. It might make more sense to use something less specific, like `Resource` or `Fuel`.

Also I agree it's sort of nitpicky but I think I'd prefer "two" over "2".

**Comment by ZephireNZ** [comment]
Just looked at the IRC logs, I agree that `setLapis()/getLapis()` is probably a bit too specific, I think I'll take the suggestion from turt2live to use `setResource()/getResource()`. I'll also update that javadoc :)

**Comment by ZephireNZ** [comment]
Little late, but pulled in f6ee29b

**Pull request closed without merging**